src/Controller/Component/CsrfComponent.php

```diff
@@ -120,12 +121,14 @@ public function implementedEvents()
      */
     protected function _setCookie(Request $request, Response $response)
     {
+        $expires = new Time($this->_config['expiry']);
```

Comment:
Will this cause issues with any old integer based config values? Or were they always broken/wrong?